org.standardnotes.standardnotes.metainfo.xml

```diff
@@ -2,10 +2,10 @@
 <component type="desktop-application">
   <id>org.standardnotes.standardnotes</id>
 
-  <name>Standard Notes</name>
-  <summary>A simple and private place for your notes, thoughts, and life's work</summary>
+  <name>Standard Notes (Unofficial)</name>
```

Comment:
There is no app listed in flathub having an `unofficial` suffix. I think it's sufficient to have the statement in the description/summary.